sonar-scanner-engine/src/main/java/org/sonar/batch/bootstrapper/Batch.java

```diff
@@ -108,26 +104,18 @@ public synchronized Batch start(boolean preferCache) {
 
   /**
    * @since 4.4
+   * @deprecated since 6.6 use {@link #execute()}
    */
+  @Deprecated
   public Batch executeTask(Map<String, String> analysisProperties, Object... components) {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)